### PR TITLE
Updated ishtob userspace to include block defines

### PR DESCRIPTION
**Pull request by ishtob**
this should make implementing other boards to my userspace easier as the common keymap locations are now centralized to the userspace.

### CODE REVIEW

File: tmk_core/common/chibios/eeprom_stm32.h [hunk 1]
```diff
@@ -48,7 +48,7 @@
     #elif defined (MCU_STM32F103RD)
         #define EEPROM_START_ADDRESS    ((uint32_t)(0x8000000 + 384 * 1024 - 2 * EEPROM_PAGE_SIZE))
     #elif defined (MCU_STM32F303CC)
-        #define EEPROM_START_ADDRESS    ((uint32_t)(0x8000000 + 250 * 1024 - 2 * EEPROM_PAGE_SIZE))
+        #define EEPROM_START_ADDRESS    ((uint32_t)(0x8000000 + 256 * 1024 - 2 * EEPROM_PAGE_SIZE))
```
Comment (drashna) [inline tmk_core/common/chibios/eeprom_stm32.h]: Could thes be made as a separate PR?

File: users/ishtob/ishtob.c [hunk 2]
```diff
@@ -43,6 +44,22 @@ bool process_record_secrets(uint16_t keycode, keyrecord_t *record) {
...
 }
 
+
+bool muse_mode = false;
+uint8_t last_muse_note = 0;
+uint16_t muse_counter = 0;
+uint8_t muse_offset = 70;
+uint16_t muse_tempo = 50;
+
+#ifdef AUDIO_ENABLE
```
Comment (drashna) [inline users/ishtob/ishtob.c]: The layer floats aren't needed, actually: 
https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

File: keyboards/planck/keymaps/ishtob/keymap.c [hunk 3]
```diff
@@ -331,12 +230,12 @@ void startup_user()
     rgblight_mode(RGB_MATRIX_CYCLE_ALL);
     #endif //RGB_matrix
     wait_ms(20); // gets rid of tick
-    PLAY_NOTE_ARRAY(tone_startup, false, 0);
+    PLAY_SONG(tone_startup);
```
Comment (drashna) [inline keyboards/planck/keymaps/ishtob/keymap.c]: Isn't needed if you define this in the config.h:
https://docs.qmk.fm/#/feature_audio

File: tmk_core/common/chibios/eeprom_stm32.h (hunk 1 above)
Comment (ishtob) [reply tmk_core/common/chibios/eeprom_stm32.h]: oh odd, I thought this was merged already

Comment (drashna) [reply tmk_core/common/chibios/eeprom_stm32.h]: You probably didn't merge/rebase then.  Should be fine, I think. Unless you wanted to fix it.  Let me know either way.

File: users/ishtob/ishtob.c (hunk 2 above)
Comment (ishtob) [reply users/ishtob/ishtob.c]: had to keep the plover floats as its called in a function later on, rest are removed

File: keyboards/planck/keymaps/ishtob/keymap.c (hunk 3 above)
Comment (ishtob) [reply keyboards/planck/keymaps/ishtob/keymap.c]: removed
